src/wp-admin/includes/class-wp-automatic-updater.php

```diff
@@ -1540,6 +1540,16 @@ protected function send_plugin_theme_email( $type, $successful_updates, $failed_
 		 */
 		$email = apply_filters( 'auto_plugin_theme_update_email', $email, $type, $successful_updates, $failed_updates );
 
+		if ( defined( 'WP_DEBUG' ) && WP_DEBUG && 'fail' === $type ) {
+			$fatal_error = get_transient( 'wp_last_fatal_error' );
+			if ( $fatal_error ) {
+				$email['body'] .= "\n\n=== LAST FATAL PHP ERROR ===\n";
```

Comment:
This should be translated: 


```suggestion
				$email['body'] .= "\n\n=== "  . __( 'LAST FATAL PHP ERROR' ) . " ===\n";
```